modules/hotmaps/1.0/config/default.yaml

```diff
@@ -0,0 +1,92 @@
+lcr-modules:
+    
+    hotmaps:
+
+        inputs:
+            # Available wildcards: {seq_type}
+            # No genome_build wildcard because only grch37 is compatible!
```

Comment:
I know it might be somewhat overwhelming but I think there is something important to address: we need to keep in some way the genome build because absence of the wildcard in this module breaks the structure of the output.
I think there is somewhat consistent and more hacky way: one option is to add the genome build wildcard as in other modules but add the check at the top of the snakefile so if this wildcard is something different then it stops and does not execute (there is an example in hmftools or gridss module). The other option is to just add it to the path of the 99-outputs  and have it hard-coded

Reply:
I added the genome build wildcard and hardcoded it to grch37!